apps/web-giddh/src/app/subscription/subscription.component.ts

```diff
@@ -1,769 +1,372 @@
+            }
+        }), takeUntil(this.destroyed$));
     }
 
     /**
-     * Initializes the component by subscribing to route parameters and fetching subscription data.
-     * Navigates to the subscription page upon subscription cancellation.
+     * Copy Authkey to Clipboard
      *
      * @memberof SubscriptionComponent
      */
-    public ngOnInit(): void {
-        document.body?.classList?.add("subscription-page");
-        this.initForm();
-        this.getAllSubscriptions(false);
-
-        /** Get Discount List */
-        this.subscriptionList$.pipe(takeUntil(this.destroyed$)).subscribe(response => {
-            if (response) {
-                this.subscriptions = response?.body?.results;
-                this.dataSource = new MatTableDataSource<any>(response?.body?.results);
-                if (this.dataSource?.filteredData?.length || this.subscriptionListForm?.controls['companyName']?.value ||
-                    this.subscriptionListForm?.controls['billingAccountName']?.value ||
-                    this.subscriptionListForm?.controls['subscriberName']?.value ||
-                    this.subscriptionListForm?.controls['countryName']?.value ||
-                    this.subscriptionListForm?.controls['planName']?.value ||
-                    this.subscriptionListForm?.controls['status']?.value ||
-                    this.subscriptionListForm?.controls['duration']?.value) {
-                    this.showData = true;
-                } else {
-                    this.showData = false;
-                }
-                this.dataSource.paginator = this.paginator;
-                this.subscriptionRequestParams.totalItems = response?.body?.totalItems;
-            } else {
-                this.dataSource = new MatTableDataSource<any>([]);
-                this.subscriptions = [];
-                this.showData = false;
-                this.subscriptionRequestParams.totalItems = 0;
-            }
-        });
+    public toggleIsCopied(): void {
+        this.isCopied = true;
+        this.clipboardService.copyFromContent(this.userAuthKey);
+        setTimeout(() => {
+            this.isCopied = false;
+        }, 3000);
+    }
 
-        this.componentStore.activeCompany$.pipe(takeUntil(this.destroyed$)).subscribe(response => {
-            if (response && this.activeCompany?.uniqueName !== response?.uniqueName) {
-                this.activeCompany = response;
-            }
-        });
+    public ngOnInit() {
+        document.querySelector('body').classList.add('setting-sidebar-open');
 
-        this.cancelSubscription$.pipe(takeUntil(this.destroyed$)).subscribe(response => {
-            if (response) {
-                this.router.navigate(['/pages/subscription']);
-            }
+        this.breakPointObservar.observe([
+            '(max-width:767px)'
+        ]).pipe(takeUntil(this.destroyed$)).subscribe(result => {
+            this.isMobileScreen = result.matches;
```

Comment:
what's the use of isMobileScreen here?

Reply:
Resolved but this is an old subscription